apps/mobile/src/libs/components/task-action-button/task-action-button.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+
+import { Icon, Pressable, Text, View } from "~/libs/components/components";
+import { BaseColor } from "~/libs/enums/enums";
+import { useMemo } from "~/libs/hooks/hooks";
+import { globalStyles } from "~/libs/styles/styles";
+
+import { styles } from "./styles";
+
+type Properties = {
+	isDisabled?: boolean;
+	isLabel?: boolean;
+	label: string;
+	onPress?: () => void;
+	type: "complete" | "skip";
+};
+
+const ICON_SIZE = 15;
+
+const TaskActionButton: React.FC<Properties> = ({
+	isDisabled = false,
+	isLabel = false,
```

Comment:
I use this property to set the button's disabled state and apply styles for it — it may not be the good approach, but I am reusing this button both as a button and as a label for the state. 
![image](https://github.com/user-attachments/assets/e2135704-fff8-479c-9589-f2af771ec424)
![image](https://github.com/user-attachments/assets/17fb8dcf-02c3-4c0b-bd63-654a7197bb65)

Reply:
"Skipped" is not a button, so it shouldn't be a part of a button component